plugins/modules/sns_topic.py

```diff
@@ -363,17 +366,20 @@ def __init__(self,
         self.attributes_set = []
 
     def _create_topic(self):
-        attributes = {'FifoTopic': 'false'}
         tags = []
```

Comment:
```suggestion
        attributes = {}
        tags = []
```

Attributes still needs to be initialised